src/coreclr/vm/interpexec.cpp

```diff
@@ -532,23 +532,23 @@ void InterpExecMethod(InterpreterFrame *pInterpreterFrame, InterpMethodContextFr
                     ip += 3;
                     break;
                 case INTOP_CONV_R4_I4:
-                    LOCAL_VAR(ip[1], float) = HCCALL1(JIT_Lng2Flt, (int64_t)LOCAL_VAR(ip[2], int32_t));
+                    LOCAL_VAR(ip[1], float) = JIT_Lng2Flt((int64_t)LOCAL_VAR(ip[2], int32_t));
                     ip += 3;
                     break;
                 case INTOP_CONV_R4_I8:
-                    LOCAL_VAR(ip[1], float) = HCCALL1(JIT_Lng2Flt, LOCAL_VAR(ip[2], int64_t));
+                    LOCAL_VAR(ip[1], float) = JIT_Lng2Flt(LOCAL_VAR(ip[2], int64_t));
```

Comment:
```suggestion
                    LOCAL_VAR(ip[1], float) = (float)LOCAL_VAR(ip[2], int64_t);
```

Reply:
It seems we can do the same for `JIT_Lng2Dbl`, as it is also just a cast to double.